### DISCUSSION TITLE
object opening icons (>) in the object tree are too insensitive

### DISCUSSION THREAD

**Discussion by heradv** [dbeaver/dbeaver#14745] [Ideas]
when i click on an object opening icon (>) in the object tree, then i have to keep in mind that the sensing area for opening and the area of the actual icon is not the same. the ">" icon is approximately 4 pixels wide and when i click on the first 2 pixels then nothing happens except highlight the object. i noticed that the 2nd half of the icon and most of the space before the object name is sensitive for opening, but i think its not intuitive and very annoying for a beginner user. is it would be possible to change this behaviour to the whole icon could open the object?

**Comment by ShadelessFox** [comment]
Hello @heradv,

It depends on the operating system you're using. 

What's your operating system? Can you provide a video showing the problem?
